mmrazor/models/quantizers/base.py

```diff
@@ -30,3 +35,54 @@ def swap_ff_with_fxff(self, model):
         for name in modules_to_swap:
             del model._modules[name]
             model._modules[name] = torch.ao.nn.quantized.FXFloatFunctional()
+
+    def del_fakequant(self, prepared):
+        prepared = del_fakequant_before_module(
+            prepared, self.module_del_prev_fakequant, inplace=True)
```

Comment:
```suggestion
        extra_module_del_prev_fakequant = self.extra_redundant_fakequants.get('module_del_prev_fakequant', tuple())
        prepared = del_fakequant_before_module(
            prepared, self.module_del_prev_fakequant + extra_module_del_prev_fakequant , inplace=True)
```
There may be some redundant fakequant due to the model, not the backend